client/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This [PR ](https://github.com/FightPandemics/FightPandemics/pull/258) configures ESlint in `client/.eslintrc.json`. We also have `backend/.eslintrc.json`. I think to stay consistent, maybe we could wait for that PR to merge, and then add these rules on top of that. What do you think?

Reply:
Similar to my comment on the other ESLint changes for the frontend, shouldn't we use the create-react-app way to change ESLint config? https://github.com/FightPandemics/FightPandemics/pull/258#discussion_r422545352
> CRA isn't fully configurable unless "ejected", you can read more about this here: https://create-react-app.dev/docs/available-scripts#npm-run-eject but we probably want to avoid that unless absolutely necessary because it also has disadvantages.
> 
> For ESLint there is a setting that can be changed and then we can add some additional ESLint settings, that will be added to the settings that are provided by CRA, see https://create-react-app.dev/docs/advanced-configuration
> 
> So I think that's the way to go, and to stay within the scope of this PR you should probably only add what's needed, which is the `"sort-imports"` rule and `"sort-keys"`

Now I'm not sure how that works with CRA but indeed seems likely that it still needs to extend the `react-app` config.

Also didn't know that this needs the `import` plugin, or it might provide some additional benefits?

Reply:
@robinv85  I wasn't aware of that, I guess I could wait until the other PR is merged and then just apply this rule to it , and yea according to the guide was posted , it looked like that plugin was needed , cause the rule is part of that package.